### PR TITLE
Use native bind instead of own fix:#7408

### PR DESCRIPTION
**Pull request by dmitry-vakhnenko**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch for v2.x (or to a previous version branch), _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] All tests are passing: https://github.com/vuejs/vue/blob/dev/.github/CONTRIBUTING.md#development-setup
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: packages/weex-vue-framework/factory.js [hunk 1]
```diff
@@ -175,23 +175,6 @@ var hyphenate = cached(function (str) {
...
-        ? fn.apply(ctx, arguments)
-        : fn.call(ctx, a)
-      : fn.call(ctx)
-  }
-  // record original fn length
-  boundFn._length = fn.length;
-  return boundFn
-}
-
 /**
```
Comment (Hanks10100) [inline packages/weex-vue-framework/factory.js]: This file is generated by `npm run build:weex`, no need to patch this.

Comment (HerringtonDarkholme) [reply packages/weex-vue-framework/factory.js]: Other than code patch, will using native bind benefit weex? I wonder.

**Comment by dmitry-vakhnenko** [comment]
> Why removing over reimplementing the bind utility as doing fn.bind(context)?
>
> Keeping and changing would allow for a central place to revert, change or optimize based on browser if that ever matters.

I do this because own implementation is slower than native -- https://jsperf.com/vue-bind-perf
I didn't know how to check perfomance for select implementation.

hm, in most modern browsers it will be good change, but i didn't know how it be work in weex. Should i save own boundFn implementation?

**Comment by OEvgeny** [comment]
Maybe just reimplement current `bind` as something like:
```js
function bind (fn, ctx) {
  return fn.bind(ctx)
}
```
as an easy to revert solution for the start?

File: packages/weex-vue-framework/factory.js (hunk 1 above)
Comment (Hanks10100) [reply packages/weex-vue-framework/factory.js]: In my opinion, using js tricks to optimise native APIs is not a good solution. JS engines are changing all the time, this trick may not always efficient.

For Weex, it's possible to optimise the JS engine directly if the native bind has performance issues.

**Pull request closed without merging**